### PR TITLE
JOIN w/o USING or ON clause

### PR DESCRIPTION
**Pull request by avbelov23**
JOIN can be without USING or ON clause.
Example: MySQL

### CODE REVIEW

File: lib/sqli/sqli_parser.y [hunk 1]
```diff
@@ -360,7 +360,7 @@ alias_opt:
 table_ref:
           func alias_opt
         | colref_exact alias_opt
-        | '('[u1] select ')'[u2] alias_opt {YYUSE($u1); YYUSE($u2);}
+        | select_parens alias_opt
```
Comment (sorc1) [inline lib/sqli/sqli_parser.y]: Why did you change that? Does a table reference support SELECT without parentheses?

Comment (avbelov23) [reply lib/sqli/sqli_parser.y]: This is a mistake. Thank you very much!

**Comment by avbelov23** [comment]
DONE

**Comment by sorc1** [comment]
The changes are now in master branch. Thanks!
